### DISCUSSION TITLE
Explanation of parentheses

### DISCUSSION THREAD

**Discussion by erquhart** [expo/router#561] [Q&A]
Hi all,

I'm trying to understand the role of parentheses in route names, eg., `(app)`/`(auth)`. I see mention of [route groups](https://expo.github.io/router/docs/guides/auth) in the authentication guide, and I'm guessing these parentheses are related to that concept, but this syntax isn't explained head on and I'd like to understand it.

Is it just a pattern like the `use` prefix on hooks or are parens wrapped folders treated differently than others?

**Comment by erquhart** [comment]
Found it in the docs: https://expo.github.io/router/docs/features/routing#groups

**Reply by g-otn** [reply]
GitHub redirect breaks anchor, here's the direct link: https://docs.expo.dev/router/layouts/#groups
